zeebe/docker/utils/startup.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh -xeu
 
 HOST=$(hostname -i)
+ZEEBE_SET_SEED_ENV_VARS=${ZEEBE_SET_SEED_ENV_VARS:-true}
```

Comment:
Let's call it `ZEEBE_DOCKER_SET_SEED_ENV_VARS` so it's clear it's used for Docker.

Reply:
✅